hack/run-e2e-kind.sh

```diff
@@ -157,7 +179,9 @@ case ${E2E_TYPE} in
     ;;
 esac
 
+generate-log
+
 if [[ $? -ne 0 ]]; then
-  generate-log
+
```

Comment:
Will generate-log be put here?

Reply:
the modifications made in run-e2e-kind.sh are temporary and were used for debugging
I will revert these changes in this PR and create a new one